app/representers/api/auth/podcast_import_representer.rb

```diff
@@ -0,0 +1,28 @@
+# encoding: utf-8
+
+class Api::Auth::PodcastImportRepresenter < Api::BaseRepresenter
+  property :id, writeable: false
+  property :status, writeable: false
+  property :url
+
+  link rel: :account, writeable: true do
+    {
+      href: api_account_path(represented.account),
+      title: represented.account.name,
+      profile: model_uri(represented.account)
+    } if represented.id && represented.account_id
+  end
+  embed :account, class: Account, decorator: Api::Min::AccountRepresenter
+
+  link rel: :series, writeable: true do
```

Comment:
I was envisioning we might want someone to be able to import into an existing series, but yeah, could be readonly.